styles/login-page.css

```
@@ -121,7 +121,7 @@
 }
 
 .login-headline {
-  font-family: 'Outfit', 'Plus Jakarta Sans', system-ui, sans-serif;
+  font-family: 'Outfit', 'Plus Jakarta Sans', 'Noto Sans JP', 'Segoe UI', 'Microsoft YaHei', 'Meiryo', 'Apple SD Gothic Neo', 'PingFang SC', 'Hiragino Sans', system-ui, sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)